roles/node_exporter/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: 'set node exporter repository'
+  yum_repository:
+    name: prometheus-rpm_release
+    baseurl: https://packagecloud.io/prometheus-rpm/release/el/$releasever/$basearch
+    description: prometheus-rpm_release
+    enabled: True
```

Comment:
Maybe keep disabled by default and explicitly enable when installing `node_exporter`?

Reply:
For update sake ? Talking to @ballot-scality, there is some plugin to install yum-version-lock. Might be better

Reply:
Update would all of a sudden install 0.16 :) I'm not too fond of version-locking myself. I believe Kubespray even uses a separate yum.conf for Docker installs to keep things separated. You can enable repos on `yum` invocation (so likely also through the `yum` module)

Reply:
repo docker are enabled (and yes install docker-ce will install 18.03, which is bad).
We must avoid doing the same mistake of kubespray. Version locking could fix docker problem (because disabling docker repo after kubespray will not prevent kubespray itself to reenable it).
It will also allow some upgrade and prevent what we have on the ring multiple times: "oh the package is now on epel so the system decide to take the version of epel instead of our repo."

Reply:
Fair enough :smiley: